src/app/app-initializer.ts

```diff
@@ -2,21 +2,23 @@ import { select, Store } from '@ngrx/store';
 import { take } from 'rxjs/operators';
 import { AppConstants } from './app.constants';
 import * as fromApp from './store';
+import * as fromSelectors from './store/selectors/app.selectors';
 
 export function initApplication(store: Store<fromApp.State>): VoidFunction {
   return () => new Promise(resolve => {
     store.dispatch(new fromApp.StartAppInitilizer());
     store.dispatch(new fromApp.LoadFeatureToggleConfig([AppConstants.FEATURE_NAMES.feeAccount,
                                                         AppConstants.FEATURE_NAMES.editUserPermissions,
                                                         AppConstants.FEATURE_NAMES.unassignedCases]));
+
     store.pipe(
-      select((state: fromApp.State) => state.appState),
+      select(fromSelectors.getAppState),
```

Comment:
What issue does this resolve?

Reply:
resolves `ERROR: 'Unhandled Promise rejection:', 'Cannot read property 'subscribe' of undefined'`